src/main/scala/cromwell/engine/WdlStandardLibraryImpl.scala

```diff
@@ -126,14 +131,22 @@ trait WdlStandardLibraryImpl extends WdlStandardLibraryFunctions {
   override def write_map(params: Seq[Try[WdlValue]]): Try[WdlFile] = writeToTsv(params, classOf[WdlMap])
   override def write_object(params: Seq[Try[WdlValue]]): Try[WdlFile] = writeToTsv(params, classOf[WdlObject])
   override def write_objects(params: Seq[Try[WdlValue]]): Try[WdlFile] = writeToTsv(params, classOf[WdlArray])
-  override def writeTempFile(path: String,prefix: String,suffix: String,content: String): String = interface.writeTempFile(path, prefix, suffix, content)
+  override def writeTempFile(path: String,prefix: String,suffix: String,content: String): String = {
+    // This may be called multiple times with the same inputs.  Calling this twice with the same
+    // parameters should yield the same return value.
+    val fullPath = s"$path/$prefix${content.md5Sum}$suffix".toAbsolutePath
+    if (!fullPath.exists) {
+      fullPath.write(content)
+    }
+    fullPath.toString
+  }
 
-  override def glob(path: String, pattern: String): Seq[String] = interface.glob(path, pattern)
+  override def glob(path: String, pattern: String): Seq[String]
```

Comment:
Oops - (Besides, "overridden back to abstraction" does not sound like a cool thing to experience :s)